modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -179,10 +300,17 @@ protected function render() {
 			$this->print_child( $index );
 			$item_content = ob_get_clean();
 
-			$items_title_html .= "\t<details {$title_render_attributes}>";
-			$items_title_html .= "\t\t<summary class='e-n-accordion-item-title'><{$tag}>{$item_title}</{$tag}></summary>";
-			$items_title_html .= "\t\t{$item_content}";
-			$items_title_html .= "\t</details>";
+			ob_start();
+			?>
+				<details <?php echo $title_render_attributes // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
+					<summary class='e-n-accordion-item-title'>
+						<span class='e-n-accordion-item-title-text'><?php echo "<$titleHTMLTag> $item_title </$titleHTMLTag>" // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></span>
+						<?php echo $icons // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
+					</summary>
```

Comment:
Maybe try using clearer variable names. What is the meaning of `icons` and `icon_content`.